### ISSUE TITLE
оптимизировать process.processIterator

### ISSUE THREAD

**Issue by ehlyzov** [akil-io/dpe#12]
Сейчас следующая пачка заданий берется только после того, как полностью завершилась предыдущая пачка. Будет лучше, если задания будут выдаваться как только завершилось хотя бы одно.

**Comment by FenixDeveloper** [comment]
Да, этого изначально хотелось, но пока не реализовано ввиду более приоритетных задач. Пулл-реквест приветствуется. На текущий момент по планам исправить к 1.5, так как решается архитектурно проще.
